controllers/storagecluster/consumer_grpc_error_handling.go

```diff
@@ -0,0 +1,44 @@
+package storagecluster
+
+import (
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+	corev1 "k8s.io/api/core/v1"
+
+	ocsv1 "github.com/red-hat-storage/ocs-operator/api/v1"
+)
+
+// handleOnBoardConsumerErrors returns the bool which tells to requeue or not
+func (r *StorageClusterReconciler) handleOnBoardConsumerErrors(instance *ocsv1.StorageCluster, err error) bool {
```

Comment:
Why is this a function and not part of onBoardConsumer?

Reply:
I think It makes it easy to read and maintain the code. Now we have 4 func for 4 different calls for error handling.